File: AirHealthV5.Server/Domain/Models/DeviceModel.cs
using AirHealthV5.Server.Interfaces;

namespace AirHealthV5.Server.Domain.Models;

public class DeviceModel : IDevice
{
    public int Id { get; set; }
    public string DeviceId { get; set; }
    public string DeviceName { get; set; }
    public string ApiKey { get; set; }
    public Guid UserId { get; set; }
    public bool IsActive { get; set; }
    public DateTime LastUpdate { get; set; }
    public DateTime Created { get; set; }
    public int[] SensorOrder { get; set; }
    public UserModel User { get; set; }
}

File: airhealthv5.client/src/app/models/DeviceModel.ts
import {ThresholdsModel} from "./Thresholds";

export interface DeviceModel {
  id: number;
  deviceId: string;
  deviceName: string;
  apiKey: string;
  userId: string;
  isActive: boolean;
  lastUpdate: Date;
  created: Date;
  sensorOrder: number[];
  thresholds: ThresholdsModel | null;
}
